apps/user/interfaces/user.interface.ts

```diff
@@ -113,10 +113,10 @@ export interface ISendVerificationEmail {
   }
   
   export interface ICheckUserDetails {
```

Comment:
Can we reduce no. of fields as discussed earlier?

Reply:
Yes sir.

Reply:
Resolved.